cfg/cis-1.6/master.yaml

```diff
@@ -206,24 +209,30 @@ groups:
         remediation: |
           Run the below command (based on the file location on your system) on the master node.
           For example,
-          chmod 644 $schedulerkubeconfig
+          chmod 644 $KUBECONFIG
```

Comment:
I think this will not since variables sets in audit won't be saved, currently I think it will be best to edit the config file and add there more paths, It will be a good thing the add more dynamically tests for configs, but its need to be part of the code and not in the audit

Reply:
But such a way we should add an extra path each time to the config. Is it better than getting the path dynamic?